### PR TITLE
Add mission control package documentation

### PR DESCRIPTION
**Pull request by patapufate**
This PR adds the documentation of the mission control package. It also adds Groot app documentation.

### CODE REVIEW

File: mission_control/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Mission Control
+The mission control package provides an implementation of missions using behavior trees
+It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) library.  
+
+## Features
+
+- The mission control implements the interface with the Jaus Ros Bridge ROS Package.
```
Comment (hidmic) [inline mission_control/README.md:7]: @patapufate how about: 
```suggestion
- Topic interface for mission loading, execution, abortion, and monitoring, compliant with that of its predecessor (`mission_manager`)
```

File: mission_control/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+The behaviors/actions implemented were:
```
Comment (hidmic) [inline mission_control/README.md:9]: @patapufate how about 
```suggestion
- Actions supported are:
```
and then indent the list below.

File: mission_control/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+- Abort
+- Attitude Servo
+- Fix Rudder
+- Go To Waypoint
+- Payload Command
+- Set Altitude Heading
+- Set Depth Heading
```
Comment (hidmic) [inline mission_control/README.md:16]: @patapufate a one liner explaining what the action does would be great.

File: mission_control/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+- The missions can be designed using Groot
```
Comment (hidmic) [inline mission_control/README.md:18]: @patapufate I think one bullet should talk about missions being described as behavior trees in XML.

File: mission_control/README.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
```
Comment (hidmic) [inline mission_control/README.md:19]: @patapufate meta: I'd also mention it's fail safe :)

File: mission_control/doc/Groot/README.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/BehaviorTree.CPP) library. 
+
+The next behaviors/actions were implemented as tree node model (Groot Palette)
+- Abort
+- Attitude Servo
+- Fix Rudder
+- Go To Waypoint
+- Payload Command
+- Set Altitude Heading
+- Set Depth Heading
```
Comment (hidmic) [inline mission_control/doc/Groot/README.md:16]: @patapufate this is already in the README, no?

File: mission_control/doc/Groot/README.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+
+Instructions to a standalone Groot installation on Windows can be found [here](https://github.com/BehaviorTree/Groot/blob/master/qt_installer_howto.md).
+
+## Instructions
+
+### Creating new Missions
+
+For creating mission with the custom behaviors, the tree node model must be loaded.
+
+- Load the Tree Node Model to get the custom behavior/actions from the toolbar:
```
Comment (hidmic) [inline mission_control/doc/Groot/README.md:39]: @patapufate I'd stick with BehaviorTree.CPP terminology: 
```suggestion
- Load the Tree Node Model to get the custom actions from the toolbar:  
```

File: mission_control/doc/Groot/README.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+## Instructions
+
+### Creating new Missions
+
+For creating mission with the custom behaviors, the tree node model must be loaded.
+
+- Load the Tree Node Model to get the custom behavior/actions from the toolbar:  
+![Toolbar](Images/toolbar.png "Toolbar")
+- Select the icon to load a pallette. The palette can be found in the [mission control package](https://github.com/ChrisScianna/ROS-Underwater-RnD/tree/master/mission_control/groot_palette)
```
Comment (hidmic) [inline mission_control/doc/Groot/README.md:41]: @patapufate nit: this is the `mission_control` package. A relative link to the pallette would be best.

File: mission_control/README.md (hunk 1 above)
Comment (patapufate) [reply mission_control/README.md:7]: Fixed

File: mission_control/README.md (hunk 3 above)
Comment (patapufate) [reply mission_control/README.md:16]: Fixed

File: mission_control/README.md (hunk 5 above)
Comment (patapufate) [reply mission_control/README.md:19]: Done

File: mission_control/doc/Groot/README.md (hunk 6 above)
Comment (patapufate) [reply mission_control/doc/Groot/README.md:16]: Fixed

File: mission_control/doc/Groot/README.md (hunk 7 above)
Comment (patapufate) [reply mission_control/doc/Groot/README.md:39]: Fixed

File: mission_control/doc/Groot/README.md (hunk 8 above)
Comment (patapufate) [reply mission_control/doc/Groot/README.md:41]: Done

File: mission_control/README.md (hunk 4 above)
Comment (patapufate) [reply mission_control/README.md:18]: Done

File: mission_control/README.md (hunk 2 above)
Comment (patapufate) [reply mission_control/README.md:9]: Done

File: mission_control/README.md [hunk 9]
```diff
@@ -4,18 +4,20 @@ It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/
...
+&nbsp; - **Abort**. Set the fins to surface and Thruster Velocity to 0 RPM.  
+&nbsp; - **Attitude Servo**. It sends the command to actuator setting the roll, pitch, yaw and speed parameters.  
+&nbsp; - **Fix Rudder**. It sends the command to actuators setting the rudder, depth and speed.  
+&nbsp; - **Go To Waypoint**. It sends command to autopilot to go to altitude, latitude, longitude and speed.  
+&nbsp; - **Payload Command**. It sends a string command to Payload  
+&nbsp; - **Set Altitude Heading**. It sends command to actuator setting the altitude, heading and speed.  
+&nbsp; - **Set Depth Heading**. It sends command to actuator setting the depth, heading and speed.  
 
 - The missions can be designed using Groot
+- The mission are described as behavior trees in XML
```
Comment (hidmic) [inline mission_control/README.md:19]: @patapufate nit: 
```suggestion
-Mission are described as behavior trees in XML.
```

File: mission_control/README.md [hunk 10]
```diff
@@ -4,18 +4,20 @@ It is compliant with the the [BehaviorTree.CPP](https://github.com/BehaviorTree/
...
+- Abort in the event of unexpected anomalies the vehicle will undertake an abort behaviour going to surface
```
Comment (hidmic) [inline mission_control/README.md:20]: @patapufate nit:
```suggestion
- Abort missions and go to surface in the event of unexpected anomalies (e.g. system faults)
```

**Review by hidmic — APPROVED** [review]

File: mission_control/README.md (hunk 9 above)
Comment (patapufate) [reply mission_control/README.md:19]: Fixed

File: mission_control/README.md (hunk 10 above)
Comment (patapufate) [reply mission_control/README.md:20]: Fixed

**Pull request merged**